Tasks/AzurePowerShellV4/AzurePowerShell.ps1

```diff
@@ -10,6 +10,8 @@ $__vsts_input_errorActionPreference = Get-VstsInput -Name errorActionPreference
 $__vsts_input_failOnStandardError = Get-VstsInput -Name FailOnStandardError
 $targetAzurePs = Get-VstsInput -Name TargetAzurePs
 $customTargetAzurePs = Get-VstsInput -Name CustomTargetAzurePs
+$input_pwsh = Get-VstsInput -Name 'pwsh'
```

Comment:
Please validate the task before check in.

Reply:
Done